junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -80,6 +81,24 @@ static String parameterTypesAsString(Method method) {
 		return '(' + ClassUtils.nullSafeToString(Class::getSimpleName, method.getParameterTypes()) + ')';
 	}
 
+	/**
+	 * Description //TODO.
+	 *
+	 * @param testClass description
+	 * @return description
+	 */
+	static DisplayNameGenerator getDisplayNameGenerator(Class<?> testClass) {
+
+		if (testClass == Standard.class) {
+			return Standard.standardGenerator;
+		}
+		if (testClass == ReplaceUnderscores.class) {
+			return ReplaceUnderscores.replaceUnderscoresGenerator;
+		}
```

Comment:
`IndicativeSentences` is missing here, isn't it?

Reply:
You're right since it was not used I forgot to add that, on it!

Reply:
Isn't this logic already implemented [here](https://github.com/Kazhuru/junit5/blob/issue/1596-indicative-sentences/junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/DisplayNameUtils.java#L102-L111)? Maybe it could be made a public method of `DisplayNameUtils`?

Reply:
Yeah! these class has 2 methods that I was considering to use instead of the actual new ones, it took me some a time this Friday wondering a good way to solve these needs (IMHO after Marc's feedback), 

To adjust the methods and accessibility of `DisplayNameUtils` `getDisplayNameGenerator` needed a `JupiterConfiguration` parameter because this method is more general and uses the default Display Name Generator setting, I'm not so sure how to pass that configuration or make a new one and where.

For the method `getDisplayNameGeneration` also useful because we needed to find the IndicativeSentencesGeneration Annotation and it has the class IndicativeSentences,  but it only return the first DisplayNameGenerator annotation not the annotation and the parameter information that were needed.

I thought these details were too specific to start modifying methods where other classes depended on them so I decided to do it this way. Anyway if you feel that this approach falls into the problem of duplicate code, I can try to do it using the `DisplayNameUtils` class maybe with an override to the `getDisplayNameGenerator` method.